diff --git a/lib/util/mayContainChildComponent.js b/lib/util/mayContainChildComponent.js
index 9c12846d93a699b3870fcc682b70318d549a9b11..d5bc2cbd592be3c118061d85e7b77c37f3cbc4d0 100644
--- a/lib/util/mayContainChildComponent.js
+++ b/lib/util/mayContainChildComponent.js
@@ -35,7 +35,7 @@ function mayContainChildComponent(root, componentName) {
           return true;
         }
         // Check for components with the provided name.
-        if (childNode.type === 'JSXElement' && childNode.openingElement && (0, _minimatch["default"])(elementType(childNode.openingElement), componentName)) {
+        if (childNode.type === 'JSXElement' && childNode.openingElement && (0, _minimatch.minimatch || _minimatch["default"])(elementType(childNode.openingElement), componentName)) {
           return true;
         }
         if (traverseChildren(childNode, depth + 1)) {
diff --git a/lib/util/mayHaveAccessibleLabel.js b/lib/util/mayHaveAccessibleLabel.js
index 07eb76610957454796da190d118f1b1061bbad20..b27d834836c585ef23d6110516f159cfc7c06d86 100644
--- a/lib/util/mayHaveAccessibleLabel.js
+++ b/lib/util/mayHaveAccessibleLabel.js
@@ -72,7 +72,7 @@ function mayHaveAccessibleLabel(root) {
       var name = getElementType(node.openingElement);
       var isReactComponent = name.length > 0 && name[0] === name[0].toUpperCase();
       if (isReactComponent && !controlComponents.some(function (control) {
-        return (0, _minimatch["default"])(name, control);
+        return (0, _minimatch.minimatch || _minimatch["default"])(name, control);
       })) {
         return true;
       }
